"build constraints exclude all Go files" when make build

<!--🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅

Hello there! 😄

To expedite issue processing please search open and closed issues before submitting a new one.
Existing issues often contain information about workarounds, resolution, or progress updates.

Please do not remove sections of this template when submitting your issue.  If the section is not relevant to your issue, leave the section but enter "n/a" for not applicable.

Note some sections are marked "REQUIRED" and must be provided.  Never mark required sections "n/a".

🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->


# 🐞 Bug Report

### Affected Services [**REQUIRED**]
<!-- Can you pin-point one or more EdgeX services (Data, Metadata, Command, etc...) as the source of the bug? -->
<!-- ✍️edit: --> The issue is located in: ZMQ implementation files


### Is this a regression?

<!-- Did this behavior work correctly in the previous version? -->
<!-- ✍️--> Yes, the previous version in which this bug was not present was: Jakarta


### Description and Minimal Reproduction [**REQUIRED**]
when to make build, there are something wrong:
CGO_ENABLED=1 GO111MODULE=on go build -ldflags "-X github.com/edgexfoundry/edgex-go.Version=0.0.0" -o cmd/core-data/core-data ./cmd/core-data
package github.com/edgexfoundry/edgex-go/cmd/core-data
	imports github.com/edgexfoundry/edgex-go/internal/core/data
	imports github.com/edgexfoundry/edgex-go/internal/core/data/application
	imports github.com/edgexfoundry/edgex-go/internal/core/data/container
	imports github.com/edgexfoundry/go-mod-messaging/v2/messaging
	imports github.com/edgexfoundry/go-mod-messaging/v2/internal/pkg/zeromq: build constraints exclude all Go files in /Users/leo/goproject/pkg/mod/github.com/edgexfoundry/go-mod-messaging/v2@v2.2.0-dev.6/internal/pkg/zeromq
make: *** [cmd/core-data/core-data] Error 1

-----
go env:
GO111MODULE="on"
GOARCH="arm64"

GOHOSTARCH="arm64"
GOHOSTOS="darwin"

GOOS="darwin"

## 🔥 Exception or Error
<pre><code>
<!-- If the issue is accompanied by an exception or an error, please share it below: -->
<!-- ✍️-->

</code></pre>


## 🌍  Your Environment
**Deployment Environment:**
mac m1, arm64,
GO 1.7.6
**EdgeX Version [**REQUIRED**]:**
latest

**Anything else relevant?**

--- a/internal/pkg/zeromq/client.go
+++ b/internal/pkg/zeromq/client.go
@@ -1,5 +1,5 @@
-//go:build linux
-// +build linux
+//go:build !windows
+// +build !windows
 
 //
 // Copyright (c) 2021 Intel Corporation

--- a/internal/pkg/zeromq/client_test.go
+++ b/internal/pkg/zeromq/client_test.go
@@ -1,5 +1,5 @@
-//go:build linux
-// +build linux
+//go:build !windows
+// +build !windows
 
 //
 // Copyright (c) 2021 Intel Corporation

--- a/internal/pkg/zeromq/subscriber.go
+++ b/internal/pkg/zeromq/subscriber.go
@@ -1,5 +1,5 @@
-//go:build linux
-// +build linux
+//go:build !windows
+// +build !windows
 
 //
 // Copyright (c) 2021 Intel Corporation

--- a/internal/pkg/zeromq/subscriber_test.go
+++ b/internal/pkg/zeromq/subscriber_test.go
@@ -1,5 +1,5 @@
-//go:build linux
-// +build linux
+//go:build !windows
+// +build !windows
 
 //
 // Copyright (c) 2021 Intel Corporation

